Unix/http/httpclient.c

```diff
@@ -1917,10 +1919,17 @@ Page* _CreateHttpHeader(
     int r;
     char* p;
 
-#define HTTP_HEADER_FORMAT "%s %s HTTP/1.1\r\n" \
-    "Content-Length: %d\r\n"\
-    "Connection: Keep-Alive\r\n" 
+    static const char HTTP_PROTOCOL_HEADER[]     = "HTTP/1.1";
+    static const char HTTP_PROTOCOL_HEADER_LEN = MI_COUNT(HTTP_PROTOCOL_HEADER)-1;
+
+    static const char CONTENT_LENGTH_HEADER[]     = "Content-Length: ";
```

Comment:
I think we can store string in https://github.com/Microsoft/omi/blob/master/Unix/http/httpcommon.h#L99 as a macro instead of duplicated in http/http.c and http/httpclient.c, just a suggestion.